requirements.txt

```diff
@@ -0,0 +1,2 @@
+xml2rfc==3.12.10
+archive-repo==0.0.2
```

Comment:
Add:

svgcheck
toml
pyyaml

Also, I wonder if we should also look in ./requirements.txt as well (if it exists) so that people who need extra tools can just make their own.  Or is that in Makefile.venv in a way that is awkward?

Reply:
what do you mean by 'we should also look'? makefile.venv installs them automatically.

Reply:
kramdown runs it if you include an SVG using any of the 20-ish ways that it supports.  We don't need to deal with all of those ways here (those can stay in the system), but it is a common-enough (and small enough) addition I thought it worthwhile.

Reply:
Hmm, OK. I assume it needs to be on the path when kramdown-rfc2629 is run?